js/treeSortable.js

```diff
@@ -322,24 +322,38 @@ function TreeSortable() {
                              */
                             if ($sibling.length) {
                                 const distance = getDistance($(this).get(0), $sibling.get(0));
+                                var ThisParent=$(this).getParent()?$(this).getParent():$(this).getRootChild()
+                                const distance2 = getDistance($(this).get(0), ThisParent.get(0));
```

Comment:
`distance2` is not meaning anything. Instead, you could use `parentDistance`. 
```suggestion
                                const parentDistance = getDistance($(this).get(0), thisParent.get(0));
```